lib/broccoli/ember-app.js

```diff
@@ -1127,8 +1183,11 @@ class EmberApp {
     let config = this._configTree();
     let templates = this._processedTemplatesTree();
 
+    let srcTree = experiments.MODULE_UNIFICATION ? this._processedSrcTree() : null;
```

Comment:
Perhaps move the `experiments.MODULE_UNIFICATION` check to `_processedSrcTree` just call it here?